check_graphite

```diff
@@ -54,15 +64,24 @@ def check(host, metric, warning_threshold, critical_threshold,
         auth = (user, password)
     else:
         auth = None
-
-    try:
-        r = requests.get(url, params=params, auth=auth)
-    except requests.exceptions.ConnectionError, e:
-        critical('Failed to contact Graphite server', e.message)
-    try:
-        data = r.json()
-    except json.JSONDecodeError, e:
-        critical('Could not decode JSON', r.content)
+    if grafana=="yes":
+        try:
+            r = requests.get(url, headers=auth_header, params=params, verify=False)
```

Comment:
why not just wrap this line with the grafana check? the rest is the same in both branches.